systems/console-bff/payment/resolver/getCorrespondents.ts

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
It's also related to mobile money, Double check if we really needed it here.

Reply:
I included it because Kashif mentioned that we might support SaaS fee payments via mobile money in the future.